contrib/zpages/src/main/java/io/opencensus/contrib/zpages/TraceConfigzZPageHandler.java

```diff
@@ -117,7 +117,7 @@ public void emitHtml(Map<String, String> queryMap, OutputStream outputStream) {
           QUERY_COMPONENT_MAX_NUMBER_OF_ANNOTATIONS,
           TraceParams.DEFAULT.getMaxNumberOfAnnotations(),
           QUERY_COMPONENT_MAX_NUMBER_OF_NETWORK_EVENTS,
-          TraceParams.DEFAULT.getMaxNumberOfNetworkEvents(),
+          TraceParams.DEFAULT.getMaxNumberOfMessageEvents(),
```

Comment:
Does this library need to be compatible with older versions of the API?  It might be safer to call getMaxNumberOfNetworkEvents().  The same applies to the other "contrib" libraries.

Reply:
As to the text in html, I think we will have cross-language template to take care of that so now I will just revert the change as well.